tools-public/toolpad/components/NpmChart.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { createComponent } from "@mui/toolpad/browser";
+import { LineChart, Line, CartesianGrid, XAxis, Tooltip, Legend, YAxis } from 'recharts';
+
+export interface ChartProps {
+  data: any[]
+}
+
+const colors = [
+  "#1976d2",
+  "#9c27b0",
+  "#d32f2f",
+  "#ed6c02",
+  "#2f2f2f",
+  "#2e7d32",
+];
+
+export const getPackages = (inData = []) => {
+  const packages: string[] = [];
+  if(inData && inData.length > 0) {
+    Object.keys(inData[0] ?? {}).forEach(packageName => {
+      if(packageName !== 'date') {
+        packages.push(packageName);
+      }
+    });
+  }
+
+  return packages;
+}
+
+function Chart(props: ChartProps) {
+  const { data } = props;
+  // @ts-ignore
+  let packages = getPackages(data);
+
+  packages = packages.filter(function(item) {
+    return item !== '@mui/material' && item !== '@mui/core'
+  });
+
+  return (
+    <LineChart width={600} height={300} data={data}>
```

Comment:
Perhaps this could have a [`ResponsiveContainer` ](https://github.com/mui/mui-public/blob/9fe02eab8154547833802ee2cf2bf2328767be7e/tools-public/toolpad/components/Chart.tsx#L33)?

Reply:
Added